src/arviz_base/reorg.py

```diff
@@ -467,6 +469,10 @@ def references_to_dataset(references, ds, sample_dims=None):
         sample_dims = rcParams["data.sample_dims"]
     if isinstance(sample_dims, str):
         sample_dims = [sample_dims]
+    if ref_dim is None:
+        ref_dim = ["ref_dim"]
```

Comment:
I would remove this two lines and keep the if below with `if is None`. It shouldn't happen (or happen hardly ever) but otherwise it is possible someone uses `ref_dim="ref_dim"` but 2d data and the function works and uses something else as dimension names instead of erroring out so they can correct their arguments

Reply:
Done